Documentation/migration-issues.md

```diff
@@ -1,7 +1,7 @@
 ## Getting help on migration issues
 You're using the new .NET Core tools that are MSBuild-based. You took your project.json project and ran `dotnet migrate` or migrated from Visual Studio 2017...and you maybe ran into problems. 
 
-The best way to get help is to [file an issue](https://github.com/dotnet/cli/issues/new) on this repo and we will investigate and provide help and/or fixes as part of new CLI builds. Please make sure to read the instructions below and to **add the [MIGRATION] prefix to the issue title**.
+The best way to get help is to [file an issue in the cli-migrate repo](https://github.com/dotnet/cli-migrate/issues/new) and we will investigate and provide help and/or fixes as part of new CLI builds. Please make sure to read the instructions below and to **add the [MIGRATION] prefix to the issue title**.
```

Comment:
I think we should keep these issues coming to the CLI repo. We will move them if we need to. Many times, the "migration" issues are more misunderstanding or questions that will get a better tracking at getting an answer at the CLI repo. If it is indeed a problem with the migration code, then we will move it there.